src/formatter.js

```diff
@@ -88,13 +88,18 @@ export default class Formatter {
   // Puts the appropriate wrapper around a value depending on the database
   // engine, unless it's a knex.raw value, in which case it's left alone.
   wrap(value) {
-    if (typeof value === 'function') {
-      return this.outputQuery(this.compileCallback(value), true);
-    }
     const raw = this.unwrapRaw(value);
     if (raw) return raw;
-    if (typeof value === 'number') return value;
-    return this._wrapString(value + '');
+    switch (typeof value) {
```

Comment:
looks correct place to implement this (the same place where 'as something' is parsed 👍

Reply:
A type check at the top of it? I thought it would make more sense to separate the methods, since the function is named `wrapString` (thus not looking intuitively able to handle an object). Also, there are other methods in the formatter class which call the `alias` helper function, like `outputQuery` (for aliased subqueries).

Reply:
I meant that it looks like the place where you chose to implement this feature seems correct and logical. This was not a critic of any kind :) I agree that those implementations were good to be in separate methods as you did.

Reply:
I misunderstood, sorry!